spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -26,21 +26,65 @@
     let(:policy_machine_storage_adapter) { described_class.new }
 
     describe 'find_all_of_type' do
+      it 'accepts an array parameter on a column attribute' do
+        search_uuids = ['some_uuid1', 'some_uuid2']
```

Comment:
Do we need to set the uuids specifically? Can we just create two generic objects and search for the uuids of the things we created?

Reply:
Being able to create generic objects would be ideal, but it's a change that would be a bit outside the scope of this PR I think. I changed it to at least generate uuids instead of having hardcoded strings within the specs I touched.